packages/sdk/scripts/generateTokenList.ts

```diff
@@ -0,0 +1,24 @@
+import fs, { rmdirSync } from 'fs'
+import path from 'path'
+
+import { fetchTokenList } from '../src/utils/token-list-utils'
+
+const generatedDirectory = path.join(__dirname, '..', 'src', 'generated')
```

Comment:
should this be in `/misc-constants` ?

Reply:
misc-constants imports it from src/generated